server.js

```diff
@@ -45,7 +45,7 @@ app.use('/api/search', (req, res) => {
 
   let searchUri = `${Config.indexUrl}${req.path}`
   if (!_.isEmpty(req.query)) {
-    searchUri += new URL(req.originalUrl).search
+    searchUri += new URL(`${searchUri}${req.originalUrl}`).search
```

Comment:
I'm not understanding why this is necessary.  Can you explain?

Reply:
Ah, I see. I've used this idea, but using the req uri components instead in my branch.